enhancements/network/configure-ovs-alternative.md

```diff
@@ -254,6 +258,25 @@ actions:
   More details on this can be found in the enhancement:
   https://github.com/openshift/enhancements/pull/1525
 
+
+#### UPI
+
+In UPI clusters where the deployer has the ability to do arbitrary network
+configuration before deployment, there may be no need for NMState to be
+involved at deploy-time. Instead, the deployer can do the bridge
+configuration and then touch a file to indicate that configure-ovs should
+not run (the same thing the nmstate-configuration service would do in an
+IPI scenario).
+
+We likely need to move the flag file from /etc/nmstate/openshift/applied
```

Comment:
I realize this is a bit vague. I'm open to suggestions on where this file should live, or if there's a better way to signal configure-ovs that the bridge configuration is already done.